op-node/rollup/interop/managed/system.go

```diff
@@ -117,28 +117,25 @@ func (m *ManagedMode) OnEvent(ev event.Event) bool {
 		ref := x.Ref.BlockRef()
 		m.events.Send(&supervisortypes.ManagedEvent{UnsafeBlock: &ref})
 	case engine.LocalSafeUpdateEvent:
-		m.log.Info("Emitting local safe update because of L2 block", "derivedFrom", x.DerivedFrom, "derived", x.Ref)
+		m.log.Info("Emitting local safe update because of L2 block", "derivedFrom", x.Source, "derived", x.Ref)
 		m.events.Send(&supervisortypes.ManagedEvent{DerivationUpdate: &supervisortypes.DerivedBlockRefPair{
-			DerivedFrom: x.DerivedFrom,
-			Derived:     x.Ref.BlockRef(),
+			Source:  x.Source,
+			Derived: x.Ref.BlockRef(),
 		}})
 	case derive.DeriverL1StatusEvent:
 		m.log.Info("Emitting local safe update because of L1 traversal", "derivedFrom", x.Origin, "derived", x.LastL2)
 		m.events.Send(&supervisortypes.ManagedEvent{
 			DerivationUpdate: &supervisortypes.DerivedBlockRefPair{
-				DerivedFrom: x.Origin,
-				Derived:     x.LastL2.BlockRef(),
-			},
-			DerivationOriginUpdate: &supervisortypes.DerivedBlockRefPair{
-				DerivedFrom: x.Origin,
-				Derived:     x.LastL2.BlockRef(),
+				Source:  x.Origin,
+				Derived: x.LastL2.BlockRef(),
 			},
+			DerivationOriginUpdate: &x.Origin,
```

Comment:
There's a type change here, with a `Origin` instead of a `Source`. It was already in some of the code, but we should rename that soon, because it's not the `L1Origin` that is communicated here, and that is confusing.